File: python/nilai.py
tugas = float(input("Masukkan nilai tugas : "))
uts = float (input("Masukkan nilai uts"))
uas = float (input("Masukkan nilai uas"))

nilai_akhir = (0.3 * tugas) + (0.3 * uts) (0.4 * uas)

print("Nilai Akhir = ", nilai_akhir)
if nilai_akhir>=60:
    print("Status : LULUS")
else:
    print("Status : Tidak Lulus")

File: java/nilai.java
import java.util.Scanner;

public class nilai {
    public static void main (String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Masukkan Nilai Tugas : ");
        double tugas = sc.nextDouble();
        System.out.print("Masukkan Nilai UTS : ");
        double uts = sc.nextDouble();
        System.out.print("Masukkan Nilai UAS : ");
        double uas = sc.nextDouble();

        double nilaiAkhir = (0.3 * tugas) + (0.3 * uts) + (0.4 * uas);

        System.out.println("Nilai Akhir : " + nilaiAkhir);
        if (nilaiAkhir>=60)
        System.out.println("Status : LULUS");
        else
        System.out.println("Status : TIDAK LULUS");
    }
}
